client.py

```diff
@@ -1,252 +1,144 @@
+    CompactSerializer,
+    CompactWriter,
+    CompactReader,
+)
 
 """
-This is boilerplate application that configures client to connect Hazelcast Cloud cluster.
+A sample application that configures a client to connect to a Hazelcast Viridian cluster,
+and to then insert and fetch data with SQL, thus testing that the connection to
+the Hazelcast Viridian cluster is successful.
 
-See: https://docs.hazelcast.com/cloud/python-client
+See: https://docs.hazelcast.com/cloud/get-started
 """
 
+class City:
+    def __init__(self, country:str, city:str, population:int) -> None:
+        self.country = country
+        self.city = city
+        self.population = population
 
-def city(country: str, name: str, population: int) -> HazelcastJsonValue:
-    return HazelcastJsonValue({
-        "country": country,
-        "city": name,
-        "population": population,
-    })
 
+class CitySeriazlizer(CompactSerializer[City]):
+    def read(self, reader:CompactReader):
+        city = reader.read_string("city")
+        country = reader.read_string("country")
+        population = reader.read_int32("population")
+        return City(country=country, city=city, population=population)
+    
+    def write(self, writer:CompactWriter, obj:City):
+        writer.write_string("country", obj.country)
+        writer.write_string("city", obj.city)
+        writer.write_int32("population", obj.population)
 
-def country(code: str, name: str) -> HazelcastJsonValue:
-    return HazelcastJsonValue({
-        "isoCode": code,
-        "country": name,
-    })
+    def get_type_name(self):
+        return "city"
 
+    def get_class(self):
+        return City
 
-def map_example(client: hazelcast.HazelcastClient):
-    """
-    This example shows how to work with Hazelcast maps.
-    """
-    cities = client.get_map("cities").blocking()
-    cities.put("1", city("United Kingdom", "London", 9_540_576))
-    cities.put("2", city("United Kingdom", "Manchester", 2_770_434))
-    cities.put("3", city("United States", "New York", 19_223_191))
-    cities.put("4", city("United States", "Los Angeles", 3_985_520))
-    cities.put("5", city("Turkey", "Ankara", 5_309_690))
-    cities.put("6", city("Turkey", "Istanbul", 15_636_243))
-    cities.put("7", city("Brazil", "Sao Paulo", 22_429_800))
-    cities.put("8", city("Brazil", "Rio de Janeiro", 13_634_274))
-    map_size = cities.size()
-    print(f"'cities' map now contains {map_size} entries.")
-    print("-"*20)
-
-
-def sql_example(client: hazelcast.HazelcastClient):
-    sql_service = client.sql
-    create_mapping_for_capitals(sql_service)
-    clear_capitals(sql_service)
-    populate_capitals(sql_service)
-    select_all_capitals(sql_service)
-    select_capital_names(sql_service)
-
-
-def create_mapping_for_capitals(sql_service: hazelcast.client.SqlService):
-    print("Creating a mapping...")
-    # See: https://docs.hazelcast.com/hazelcast/5.1/sql/mapping-to-maps
+
+def create_mapping(client: hazelcast.client):
```

Comment:
```suggestion
def create_mapping(client: hazelcast.client.HazelcastClient):
```

Or, you can import the HazelcastClient at the top and re-use it here

Reply:
Fixed at https://github.com/hazelcast/hazelcast-cloud-python-sample-client/pull/14/commits/b4aa24d800baa69a3b29b2cde354a3e5172860d5